src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
++++
+title = "How to deploy Istio in your cluster"
+description = "Tutorial on how to deploy Istio in the differnet providers offered by Giant Swarm."
+date = "2018-09-27"
+type = "page"
+weight = 30
+tags = ["tutorial"]
++++
+
+# Preparation
+
+Nowadays, the recommended way to deploy Istio is using Helm. Please ensure the Kubernetes package manager is installed and running in your cluster. Go to the official [docs](https://docs.helm.sh/using_helm#install-helm) in case you need to install it.
+
+Now, it is time to [download the Istio project package](https://github.com/istio/istio/releases/) and include the command line interface in your path. After having it in your local uncompressed, enter in the project folder and make sure the `/bin` directory is under your system path.
+
+Verify you are ready running these commands
+```
+# Check helm client and tiller are running
+$ helm version
+Client: &version.Version{SemVer:"v2.10.0", GitCommit:"X", GitTreeState:"clean"}
+Server: &version.Version{SemVer:"v2.10.0", GitCommit:"X", GitTreeState:"clean"}
+
+# Check istioctl is available
+$ istioctl version
+Version: X.X.X
+...
+BuildStatus: Clean
+```
```

Comment:
I would separate the `istioctl version` part into its own code block vor better structure. Again, the comment content belongs in the normal article text bertween the code blocks.